src/components/tables/tests/TableUtils.spec.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  getNextTableSortingOrder,
+  getTableChildComponentId,
+  getTableLoadingIds,
+  convertInitialCollectionToDataById,
+  convertDataByIdToCollection,
+} from '../TableUtils';
+import { TableSortingOrder, TableChildComponent } from '../TableConstants';
+import * as _ from 'underscore';
+import * as faker from 'faker';
+
+describe('TableUtils', () => {
+  describe('getNextTableSortingOrder', () => {
+    const {
+      ASCENDING,
+      DESCENDING,
+      UNSORTED,
+    } = TableSortingOrder;
+
+    it('should return ASCENDING if the previous order was UNSORTED', () => {
+      expect(getNextTableSortingOrder(UNSORTED)).toBe(ASCENDING);
```

Comment:
Quick question, is it possible to change the default sort from UNSORTED? In UA tables if we sort on metrics, we sort descending by default (e.g., to get the values with most clicks first)

Reply:
you could initialize the table with `initialTableData` sorted in the way you want, but the icon in the table header would not be a reflection of this state for now